copyright

```diff
@@ -814,6 +814,10 @@ Copyright: Tommaso Becca
 License: CC-BY-SA-4.0
 Comment: Derived from works by ESA/Hubble & NASA (under the same license)
 
+Files: source/Utf8.*
+Copyright: 2017, 2018 by Flávio J. Saraiva
+License: GPL-3+
+
```

Comment:
Source files aren't shipped with the game, and therefore don't go in the copyright file. Source file copyright is handled by the block comment at the top.
```suggestion
```

Reply:
Uhm, I have no idea how to cancel the thumb-down..., it's just a misclick.

I think this states the copyright for the git repository but it's not very important because it shows only a first author and we can confirm all copyright holders by the each files and the changelog.

Reply:
That would be conflicting information, since source files are in fact handled by the copyright file:

https://github.com/endless-sky/endless-sky/blob/7de6f95e0d55d6f712e5be744240f6f232dd8701/copyright#L6-L8

I see no reason not to include it.

Reply:
Yes, but I had a idea and submitted #5346.